tests/conftest.py

```diff
@@ -85,7 +86,7 @@ def _create_user(_test_client):
             "/auth/signup",
             json={
                 "email_address": "test@fairhub.io",
-                "password": "Testingyeshello11!",
+                "password": "asdkj45@ksdSA",
```

Comment:
We should revert this the testing password. The mechanism for testing will be updated in the future to use a different db